99-make_reports.py

```diff
@@ -164,12 +164,17 @@ def run_report(subject, session=None):
                     fig_lh = plt.figure()
                     fig_rh = plt.figure()
 
-                    brain_lh = stc.plot(views='lat', hemi='lh',
-                                        initial_time=peak_time,
-                                        backend='matplotlib', figure=fig_lh)
-                    brain_rh = stc.plot(views='lat', hemi='rh',
-                                        initial_time=peak_time,
-                                        backend='matplotlib', figure=fig_rh)
+                    brain_lh = stc.plot(
+                        views='lat', hemi='lh',
+                        initial_time=peak_time,
+                        backend='matplotlib',
+                        subjects_dir=subjects_dir,
+                        figure=fig_lh)
+                    brain_rh = stc.plot(
+                        views='lat', hemi='rh',
+                        initial_time=peak_time,
+                        subjects_dir=subjects_dir,
+                        backend='matplotlib', figure=fig_rh)
```

Comment:
I would not had code matplotlib here. It should use matplotlib only because mayavi and pyvista are not available

Reply:
That's what we do :) You're in the `if mne.viz.get_3d_backend() is None` branch here :)